packages/ste-core/src/dispatching/SubscriptionChangeEventHandler.ts

```diff
@@ -1,27 +1,5 @@
-import { DispatcherBase } from "..";
 
 /**
  * Event handler type definition for subscription changes.
  */
-export type SubscriptionChangeEventHandler = (count: number) => void;
-
-/**
- * Dispatcher for subscription changes.
- * 
- * @export
- * @class SubscriptionChangeEventDispatcher
- * @extends {DispatcherBase<SubscriptionChangeEventHandler>}
- */
-export class SubscriptionChangeEventDispatcher extends DispatcherBase<SubscriptionChangeEventHandler>
```

Comment:
Where did this one go?

Reply:
Found it :-) it moved to the base

Reply:
Yeah I really could not work out how to get that in its own file without it ending up back in a circular import.